templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,35 @@
-1.  {!create-an-incoming-webhook.md!}
+# Zulip Discourse integration
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+Get notifications for new Discourse posts in Zulip!
 
-1.  Copy the `USERNAME` and `API KEY` - you'll need it later.
+{start_tabs}
 
-1.  Head over to the
-    [discourse-chat-integration setup instructions](https://meta.discourse.org/t/68501)
-    and complete them.
+1. {!create-an-incoming-webhook.md!}
 
-{!congrats.md!}
+1. Install the Discourse [Chat Integration][chat-integration].
 
-![Discourse chat integration](/static/images/integrations/discourse/001.png)
+1. In your **Discourse site settings**, fill the form with the following:
+
+    * `chat_integration_zulip_server`: {{ zulip_url }}
+    * `chat_integration_zulip_bot_api_key`: your [bot][bot]'s API key
+    * `chat_integration_zulip_bot_email_address`: your [bot][bot]'s email
+    * Enable `chat_integration_zulip_enabled`.
+
+[bot]: ../../#settings/your-bots
+
+1. Go to the **Plugins** tab, click on **Chat Integration**. Select
+   **Zulip**, and click **Add Channel**.
+
+1. Set **Stream** to the [channel](/help/create-a-channel) name that you'd
+   like to receive notifications in, set **Subject** to the topic name, and
+   click **Save Channel**.
```

Comment:
What about the step 10 for setting up rules on channel the [discourse integration instructions](https://meta.discourse.org/t/set-up-zulip-notifications-using-the-chat-integration-plugin/68501)?

Reply:
How much work would it be to use modern terminology in the actual integration ("channel" and "topic", not "stream" and "subject")?

Reply:
That would have to be changed on the Discourse end of the integration.